libs/blocks/merch/merch.js

```diff
@@ -185,9 +185,13 @@ export async function buildCta(el, params) {
   const service = await initService();
   const text = el.textContent?.replace(/^CTA +/, '');
   const cta = service.createCheckoutLink(context, text);
-  cta.classList.add('con-button');
-  cta.classList.toggle('button-l', large);
-  cta.classList.toggle('blue', strong);
+  if (el.href.includes('#_tcl')) {
```

Comment:
What `#_tcl` stands for? Can we use something easier like `#_inline` ?

Reply:
`#_inline` is already used for inline fragments (milo core feature)

Reply:
@mokimo good catch. But OST links should not match with fragment block as they have different patterns.
![image](https://github.com/adobecom/milo/assets/330057/70c3a7fd-5291-417d-894e-78bcb0399fa0)

Reply:
#_tcl it's like text checkout link

Reply:
Would there be an easier way to author this // make assumptions about what the buttons are? In milo usually only links that are italic or bold are transformed into CTAs https://milo.adobe.com/docs/authoring/button-mapping

Reply:
yeah, i wonder if its written down somewhere what are #_dnt and #_dnb.. then we can add our #_tcl to the list :)